tests/sles4sap/crash/cloud_destroy.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Public Cloud - Resource Cleanup
+# This module deletes resources from cloud:
+# - Free up cloud resources
+# - Avoid additional cost
+# - Clean up the environment
+# It's also implemented as a post_fail_hook to ensure resources
+# are deleted even if a test module fails.
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use mmapi 'get_current_job_id';
+use serial_terminal 'select_serial_terminal';
+
+
+sub run {
+    my ($self) = @_;
+
+    die('CSP not supported for the moment')
```

Comment:
sorry but this line does not make much sense . you getting value of variable and compare it with ... same variable hence "die" is not reachable , ever